ouroboros-consensus-cardano-tools/src/Cardano/Tools/DBAnalyser/Block/Cardano.hs

```diff
@@ -307,22 +329,31 @@ mkCardanoProtocolInfo genesisByron signatureThreshold genesisShelley genesisAlon
           babbageProtVer                 = ProtVer 7 0
         , babbageMaxTxCapacityOverrides  = TxLimits.mkOverrides TxLimits.noOverridesMeasure
         }
-      (unSTA shelleyTransition)
-      (unSTA allegraTransition)
-      (unSTA maryTransition)
-      (unSTA alonzoTransition)
-      (unSTA babbageTransition)
+      ProtocolParamsConway {
+          conwayProtVer                  = ProtVer 8 0
```

Comment:
It is crazy how many places the protocol version is hardcoded in
```suggestion
          conwayProtVer                  = ProtVer 9 0
```

Reply:
And it's also hard-coded in cardano-node as well. It's particularly confusing since we care about:

* the version in the block header (ie the version we declare we are ready to advance to)
* the version that will shut the node down (so that we have clean hard forks).